lib/npg_tracking/util/pipeline_config.pm

```diff
@@ -151,6 +151,24 @@ sub read_config {
   return $config;
 }
 
+=head2 default_study_config
+
+  Example    : $obj->study_config();
+  Description: Returns a default config, which is normally used as a fall-back
+               position when a study-specific config is not available. If the
+               default section of the study configuration file is not present,
+               an empty hash is returned.
+
+  Returntype : Hash
+
+=cut
+
+sub default_study_config {
```

Comment:
Should we use this method at line 222 ?

Reply:
Resolved with https://github.com/wtsi-npg/npg_tracking/pull/833/commits/b1b3f7ec28bb2cbf76af36642d6e9cebe8b5da83